ops/aws/terraform/network/main.tf

```diff
@@ -9,6 +9,26 @@ provider "aws" {
   region = "${var.region}"
 }
 
+# KMS Application Key (for encrypting and decrypting configurations / secrets)
+resource "aws_kms_key" "config_encryption_key" {
+  description = "Encryption key used to access application configurations."
+  tags {
+    Name = "${var.application_name}: ${var.environment} kms encryption key"
+    app  = "${var.application_name}"
+    env  = "${var.environment}"
+  }
+}
+
+resource "aws_kms_alias" "config_encryption_key" {
+  name          = "alias/${var.application_name}-${var.environment}"
+  target_key_id = "${aws_kms_key.config_encryption_key.key_id}"
+}
+
+output "aws_kms_arn" {
+  sensitive = true
```

Comment:
By default, outputs will always be, well, output, after a `terraform apply`.  By marking it `sensitive`, we ensure that it will not appear in `terraform apply`'s output, but can still be accessed via the command `terraform output aws_kms_arn`.